lib/nuts.js

```diff
@@ -235,38 +235,49 @@ Nuts.prototype.onUpdateRedirect = function(req, res, next) {
 Nuts.prototype.onUpdate = function(req, res, next) {
     var that = this;
     var fullUrl = getFullUrl(req);
+
     var platform = req.params.platform;
-    var channel = req.params.channel || '*';
+    var channel = req.params.channel || 'stable';
     var tag = req.params.version;
     var filetype = req.query.filetype ? req.query.filetype : 'zip';
 
     Q()
     .then(function() {
-        if (!tag) throw createError(400, 'Requires "version" parameter');
-        if (!platform) throw createError(400, 'Requires "platform" parameter');
+        if (!tag) {
+            throw createError(400, 'Requires "version" parameter');
+        }
+        if (!platform) {
+            throw createError(400, 'Requires "platform" parameter');
+        }
 
         platform = platforms.detect(platform);
 
         return that.versions.filter({
             tag: '>='+tag,
             platform: platform,
-            channel: channel
+            channel: channel,
+            stripChannel: true
```

Comment:
I tried to remove the `stripChannel` but I ran into other problem: I was not able to update an user on a stable version to a beta version, because of how `semver.satisfies` handle the prerelease tag. Such a nightmare..